doc/tutorial/00_first_example/index.md

```diff
@@ -24,5 +26,14 @@ changes in the language in recent years, we recommend the following resources:**
 
 # First head line
 
+tr `la` la `la`.
```

Comment:
Did you commit these changes by accident?

Reply:
:laughing: no I actually wanted to show that wrapping words in back ticks now styles them like `github` does in a grey box. But I guess it is to small in the picture anyway